app/src/components/fineTunes/FineTuneContentTabs/General/General.tsx

```diff
@@ -84,7 +92,10 @@ const General = () => {
         <FineTunePruningRules />
         <FineTuneDangerZone />
       </VStack>
-    </VStack>
+      <Box position="sticky" top={8} pl={{ base: 0, md: 8 }}>
```

Comment:
I think we should only render this conditionally if the model is actually deployed? It would be confusing to see this code block, try to run my model, and get an error that it isn't ready yet.